cdap-sentry/cdap-sentry-extension/cdap-sentry-binding/src/main/java/co/cask/cdap/security/authorization/sentry/binding/AuthBinding.java

```diff
@@ -721,8 +644,6 @@ private EntityId toEntityId(TAuthorizable tAuthorizable, @Nullable EntityId pare
         Program program = (Program) authorizable;
         assertNonNullParent(parent, Authorizable.AuthorizableType.PROGRAM);
         ApplicationId applicationId = (ApplicationId) parent;
-        // TODO: this method will be remmoved CDAP-12100
-        //noinspection ConstantConditions
         return applicationId.program(program.getProgramType(), program.getProgramName());
```

Comment:
Now that I think about it, trying to create an EntityId from a wildcard authorizable will throw and exception, no?